llvm/unittests/Support/KnownBitsTest.cpp

```diff
@@ -815,4 +815,38 @@ TEST(KnownBitsTest, ConcatBits) {
   }
 }
 
+TEST(KnownBitsTest, MulExhaustive) {
```

Comment:
Shouldn't this be called `MulLowerBitsExhaustive` or something?

Reply:
I don't think the current name is wrong, and I hope to extend it to high bits as well as low bits.